cmd/dockerd/daemon.go

```diff
@@ -508,10 +511,15 @@ func (cli *DaemonCli) initMiddlewares(s *apiserver.Server, cfg *apiserver.Config
 		s.UseMiddleware(c)
 	}
 
-	if err := validateAuthzPlugins(cli.Config.AuthorizationPlugins, cli.d.PluginStore); err != nil {
-		return fmt.Errorf("Error validating authorization plugin: %v", err)
-	}
-	cli.authzMiddleware = authorization.NewMiddleware(cli.Config.AuthorizationPlugins, cli.d.PluginStore)
+	// XXX: enabled plugins are not yet restored and running when this call is made.
```

Comment:
@riyazdf : I ran into errors here when an enabled plugin is looked up for validation, even before the plugin is restored to running state. How does `TestAuthZPluginAllowNonVolumeRequest` pass?